docs/modules/ROOT/pages/how-to-guides/proc_managed_services_onboarding.adoc

```diff
@@ -3,101 +3,72 @@ When an application is ready for release, the Development team contacts the Mana
 Additionally, if the Development team adds a new component to the application on their workspace,  they must notify the Managed Environment team. So they can run the `copy-application.sh` script again to ensure that the managed workspace has the most recent version of the application.
 
-== The `ReleaseStrategy` object
+== Creating a `ReleasePlanAdmission` object
 
 .*Prerequisites*
 
 * An existing Development and Managed workspace.
 
-* An existing `ReleasePlan` object.
+* An existing `ReleasePlan` object in the Development workspace.
 
 .*Procedures*
 
-* Create a release_strategy.yaml object by running:
+. Create a `ReleasePlanAdmission.yaml` object locally.
 
 +
-[source,]
-----
-oc apply -f releasestrategy.yaml -n managed
-----
-
-+ 
-*Example release_strategy.yaml object*
+*Example `ReleasePlanAdmission.yaml` object*
 
 +
 [source,yaml]
 ----
 apiVersion: appstudio.redhat.com/v1alpha1
-kind: ReleaseStrategy <.>
+kind: ReleasePlanAdmission <.>
 metadata:
-  name: strategy <.>
-  namespace: managed-environment <.>
+ name: sre-production <.>
+ namespace: managed <.>
 spec:
-  pipeline: release <.>
-  bundle:     quay.io/{ProductName}-release/release-pipeline:main <.>
-  policy: policy <.>
-----
-<.> The name of the resource that the Managed Environment team creates.
-<.> The name of the release strategy.
-<.> The name of the managed environment to which the system deploys an application.
-<.> The name of the pipeline that the system uses when releasing an application to a managed environment.
-<.> The URL to the release pipeline bundle.
-<.> The URL to the enterprise contract policy config file which  stores the policies and collections. The system validates the enterprise contract policies before releasing an application to production.
-
-== The ReleasePlanAdmission object
-
-.*Prerequisites*
-* An existing Development and Managed workspace.
-
-* An existing `ReleasePlan` object in the Development workspace.
-
-* An existing `ReleaseStrategy` object.
-
-.*Procedures*
-
-
-* Create a `release_plan_admission.yaml` object by running:
+ applications:
+  - demo-app <.>
+ data: <key> <.>
+ environment: sre-production <.>
+ origin: <dev-workspace> <.>
+ pipelineRef: <pipeline_ref> <.>
+ policy: <policy> <.>
+ serviceAccount: <service-account> <1.>
 
-+
-[source,]
-----
-oc apply -f releaseplanadmission.yaml -n managed
 ----
 
 +
-NOTE: The `release_plan_admission.yaml` represents the reciprocal link to the release_plan.yaml objects created by the development team.
+<.> The name of the release plan admission.
```

Comment:
This is misaligned now. Reference 1. in the source is the `kind` and 2. is `metadata.name`. So I think the `<.>` on the `kind` line should be removed?